ebuild-writing/functions/src_prepare/text.xml

```diff
@@ -31,13 +31,10 @@
 <section>
 <title>Default <c>src_prepare</c></title>
 <body>
-<p>
-Starting from EAPI=2, the src_prepare function is the appropriate area to perform
-any kind of patching and source code manipulation, instead of src_unpack.
-</p>
+
 <codesample lang="ebuild">
 src_prepare() {
-	true;
+	true
```

Comment:
Maybe use `:` if you're already changing it.

Reply:
I had considered it, but then decided against. `true` is easier to understand for a beginner than a lone `:`.